data/en/test_section_summary.json

```diff
@@ -0,0 +1,282 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.2",
+    "survey_id": "999",
+    "title": "Section Summary",
+    "description": "A questionnaire to test section summaries",
+    "theme": "default",
+    "legal_basis": "Voluntary",
+    "navigation": {
+        "visible": true
+    },
+    "sections": [{
+            "id": "property-details-section",
+            "title": "Property Details",
+            "groups": [{
```

Comment:
We should have at least one section with multiple groups in to see how it deals with that. Also to ensure the correct title is being used on the pages don't have the section and group titles being the same.